starter-code/views/index.ejs

```diff
@@ -5,6 +5,13 @@
     <link rel='stylesheet' href='/stylesheets/style.css' />
   </head>
   <body>
-    Hello world
+    <div id="container">
+      <ul>
+        <li><a href="/signup">Sign up</a></li>
+        <li><a href="/login">Login</a></li>
```

Comment:
I'm giving this bonus exercise: try to find a way to hide the `Sign up` and `Login` links if the user us already logged in. Good luck! ⚡️

Reply:
First idea that comes to mind is I'd try creating a new view. Is there a smarter way?

Reply:
You can use conditions in ejs files.

Reply:
Definitely smarter ;) I'll look into it.

Reply:
I tried adding a check for `req.session.currentUser` on the index view, but running into an issue:

I get a `req is undefined` on the view, not sure why. This happens even on accessing index ("/") after signing up, which passes through:

```javascript
if (bcrypt.compareSync(password, user.password)) {
           // Save the login in the session
           req.session.currentUser = user;
           res.redirect("/");
```

why isn't req.session.currentUser defined then?